Tests/test_PDB_PSEA.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2022 by Valentin Vareskic (valentin.vareskic@gmail.com).
+# All rights reserved. This code is part of the Biopython distribution
+# and governed by its license.  Please see the LICENSE file that should
+# have been included as part of this package.
+
+"""Tests for PDB PSEA."""
+
+import unittest
+from Bio.PDB.PSEA import run_psea, psea, psea2HEC, PSEA
+from Bio.PDB import PDBParser
+from subprocess import getoutput
+from Bio import MissingExternalDependencyError
+import os
+
+os.environ["LANG"] = "C"
+
+if "command not found" or "'psea' is not recognized" in getoutput("psea -h"):
+    raise MissingExternalDependencyError(
+        "Download and install psea from ftp://ftp.lmcp.jussieu.fr/pub/sincris/software/protein/p-sea/. Make sure that psea is on path"
```

Comment:
This link is dead for me, but I can get `p-sea` (note the dash in the name) here: http://bioserv.cbs.cnrs.fr/PSEA/DOWNLOAD/. Are these different versions?

Reply:
This is actually funny .. yeah, I agree, code should be the same!